app/models/page.rb

```diff
@@ -57,6 +57,10 @@ def meta_tags
     tag_names << plugin_names
   end
 
+  def meta_twitter_handle
+    Settings.meta_twitter_handles[language.code.to_sym]
```

Comment:
That makes sense. The good news is that I think if you're using the I18n API, it doesn't really need to be tested because it's the exact same lookup we do all over the place.